### ISSUE TITLE
Slowdown movement issue

### ISSUE THREAD

**Issue by FireWaterGasoline** [kirijiri/GalactosIntolerant#34]
Slowdown option only seems to counteract the speeding up of the planet (done by flicking) but does nothing to speed it back up to normal speed if you're slowing the planet down. Maybe this is intentional? 

Also, if you do slow it down and then try to speed it up, it treats the slow speed as the new normal speed and slows down the planet to that.

**Comment by kirijiri** [comment]
yes, right now thats intentional... the slow down is only triggered after
flicking

**Comment by kirijiri** [comment]
and yes it gets the speed at button press, which would be affected by the
flicking potentially, so thats still a bug, but we haven't figured out a
solution just yet

**Comment by disusedgenius** [comment]
Just to double check - is this still an issue?
